src/bin/openst.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+
+const ChainConfig = require('../config/chain_config');
+const OpenSTDeployer = require('../openst_deployer');
+const { version } = require('../../package.json');
+
+program
+  .version(version)
+  .name('bt')
+  .arguments('openst <config> <organization> <eip20Token>')
+  .description('An executable to setup OpenST.')
+  .action(
+    async (config, organization, eip20Token) => {
+      const chainConfig = new ChainConfig(config);
+      const openstDeployer = new OpenSTDeployer(chainConfig);
+      await openstDeployer.setupOpenst(organization, eip20Token);
+      chainConfig.write(config);
```

Comment:
Please use the `connected.run()` function to run executables that send transactions. See other executables as an example. Maybe you need to update your branch from `master`.

`connected.run()` assures you have the correct Web3 providers set up and the connection is closed afterwards.

Reply:
Done!